### ISSUE TITLE
The boost_1_76_0.tar.gz file is not in GZIP format

### ISSUE THREAD

**Issue by ravisharnagat** [software-mansion/react-native-reanimated#5542]
### Description

Getting following error on android build creation:

```
Execution failed for task ':react-native-reanimated:prepareBoost'.
13:32:31 > Could not read /home/ubuntu/workspace/Feature-Testing-D11/Feature-testing/AndroidAdhocBuildsNew_FLIP/node_modules/react-native-reanimated/android/build/downloads/boost_1_76_0.tar.gz.
13:32:31    > Not in GZIP format
```

### Steps to reproduce

Execution is failing for the step where boost is prepared, because the downloaded tar.gz file (https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.gz) is not present in gzip format. Please check the screenshot attached.
<img width="1464" alt="Screenshot 2023-12-31 at 1 55 48 PM" src="https://github.com/software-mansion/react-native-reanimated/assets/70310493/7d4b61fe-86aa-49ca-a45e-23c8b378aba1">

### Snack or a link to a repository

github.com/eternal_mercy_2/test-reanimated

### Reanimated version

3.3.0

### React Native version

0.69.1

### Platforms

Android

### JavaScript runtime

None

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Acknowledgements

Yes

**Comment by Kprashanth20** [comment]
facing the same issue. any solution?

**Comment by Ehabwazery** [comment]
same issue

**Comment by hodayaGruzm** [comment]
i have the same issue, any updates?

**Comment by Adwait5522** [comment]
Any Updates?

**Comment by Don-oneself** [comment]
same issue

**Comment by aryanagarwal15** [comment]
Same error, needs urgent attention

**Comment by Ehabwazery** [comment]
anyone found a solution ?

**Comment by abed-obaah** [comment]
any solution ye?

**Comment by ssommelet21** [comment]
Currently, the ball is here :

https://github.com/facebook/react-native/issues/42110#issuecomment-1872943368

**Comment by Linuhusainnk** [comment]
Any updates on the issue prepareBoost failed

**Comment by abed-obaah** [comment]
Still didn't fix it
I think it's a global issue

**Comment by chaimPaneth** [comment]
Jfrog redirects the boost source download to this https://landing.jfrog.com/reactivate-server/boostorg with an account reactivation notice.

Just replace

```
spec.source = { :http => 'https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2',
                  :sha256 => 'f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41' }
```
in `/node_modules/react-native/third-party-podspecs/boost.podspec` with

```
spec.source = { :http => 'https://sourceforge.net/projects/boost/files/boost/1.76.0/boost_1_76_0.tar.bz2',
                  :sha256 => 'f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41' }
```

Then you can also `patch-package react-native` for future installs.

**Comment by Temitope-Emmanuel** [comment]
> Jfrog redirects the boost source download to this https://landing.jfrog.com/reactivate-server/boostorg with an account reactivation notice.
> 
> Just replace
> 
> ```
> spec.source = { :http => 'https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2',
>                   :sha256 => 'f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41' }
> ```
> 
> in `/node_modules/react-native/third-party-podspecs/boost.podspec` with
> 
> ```
> spec.source = { :http => 'https://sourceforge.net/projects/boost/files/boost/1.76.0/boost_1_76_0.tar.bz2',
>                   :sha256 => 'f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41' }
> ```
> 
> Then you can also `patch-package react-native` for future installs.

I wasn't able to get it to work, has anyone tried this ?

**Comment by chaimPaneth** [comment]
@Temitope-Emmanuel 

It worked for me. Make sure to run `rm -rf Pods/ Podfile.lock` before hand. Also if your using M1 mac run first `bundle install` and `bundle exec pod install` instead of just pod install.

**Comment by Temitope-Emmanuel** [comment]
> @Temitope-Emmanuel 
> 
> It worked for me. Make sure to run `rm -rf Pods/ Podfile.lock` before hand. Also if your using M1 mac run first `bundle install` and `bundle exec pod install` instead of just pod install.

It's my android build that's failing, it's the same error message that's why I'm on this thread

**Comment by abed-obaah** [comment]
has anyone solved it yet?. the solutions provided aren't fixing it

**Comment by ravisharnagat** [comment]
For Android, one quick workaround is to create patch-package for package react-native-reanimated.

Just replace
```
    task downloadBoost(dependsOn: resolveBoost, type: Download) {
        def transformedVersion = BOOST_VERSION.replace("_", ".")
        def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
        def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
        if (REACT_NATIVE_MINOR_VERSION < 69) {
            srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
        }
        src(srcUrl)
        onlyIfNewer(true)
        overwrite(false)
        dest(new File(downloadsDir, artifactLocalName))
    }
```

in `/Users/ravisharnagat/dream11/d11-react-native/node_modules/react-native-reanimated/android/build.gradle` with 

```
        task downloadBoost(dependsOn: resolveBoost, type: Download) {
        def transformedVersion = BOOST_VERSION.replace("_", ".")
        def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
        
        def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"
        if (REACT_NATIVE_MINOR_VERSION < 69) {
            srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
        }
        src(srcUrl)
        onlyIfNewer(true)
        overwrite(false)
        dest(new File(downloadsDir, artifactLocalName))
    }
```

This should fix the android build issue for timebeing. @Temitope-Emmanuel

**Comment by abed-obaah** [comment]
> For Android, one quick workaround is to create patch-package for package react-native-reanimated.
> 
> Just replace
> 
> ```
>     task downloadBoost(dependsOn: resolveBoost, type: Download) {
>         def transformedVersion = BOOST_VERSION.replace("_", ".")
>         def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
>         def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
>         if (REACT_NATIVE_MINOR_VERSION < 69) {
>             srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
>         }
>         src(srcUrl)
>         onlyIfNewer(true)
>         overwrite(false)
>         dest(new File(downloadsDir, artifactLocalName))
>     }
> ```
> 
> in `/node_modules/react-native-reanimated/android/build.gradle` with
> 
> ```
>         task downloadBoost(dependsOn: resolveBoost, type: Download) {
>         def transformedVersion = BOOST_VERSION.replace("_", ".")
>         def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
>         
>         def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"
>         if (REACT_NATIVE_MINOR_VERSION < 69) {
>             srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
>         }
>         src(srcUrl)
>         onlyIfNewer(true)
>         overwrite(false)
>         dest(new File(downloadsDir, artifactLocalName))
>     }
> ```
> 
> This should fix the android build issue for timebeing. @Temitope-Emmanuel


what  about for managed workflow

**Comment by 247Sam** [comment]
> For Android, one quick workaround is to create patch-package for package react-native-reanimated.
> 
> Just replace
> 
> ```
>     task downloadBoost(dependsOn: resolveBoost, type: Download) {
>         def transformedVersion = BOOST_VERSION.replace("_", ".")
>         def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
>         def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
>         if (REACT_NATIVE_MINOR_VERSION < 69) {
>             srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
>         }
>         src(srcUrl)
>         onlyIfNewer(true)
>         overwrite(false)
>         dest(new File(downloadsDir, artifactLocalName))
>     }
> ```
> 
> in `/node_modules/react-native-reanimated/android/build.gradle` with
> 
> ```
> task downloadBoost(dependsOn: resolveBoost, type: Download) {
>         def transformedVersion = BOOST_VERSION.replace("_", ".")
>         def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
>         
>         def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"
>         if (REACT_NATIVE_MINOR_VERSION < 69) {
>             srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
>         }
>         src(srcUrl)
>         onlyIfNewer(true)
>         overwrite(false)
>         dest(new File(downloadsDir, artifactLocalName))
>     }
> ```
> 
> This should fix the android build issue for timebeing. @Temitope-Emmanuel

This worked for me. Make sure to delete the previously downloaded file `boost_1_76_0.tar.gz` in `react-native-reanimated/android/build/downloads`

**Comment by Dhinakrish** [comment]
@ravisharnagat Thank for your solution Its worked for me to generate Android build

**Comment by clifford-karta** [comment]
Does anyone able to solve this issue? I'm having issue during Ci/CD action build in Github.

**Comment by Linuhusainnk** [comment]
Is it resolved or apply patch, which would be better ?

**Comment by chandresh8187** [comment]
same issue

**Comment by ssommelet21** [comment]
> @ravisharnagat thank you it works for me... i'm still confused abut GZIP format in reanimated...... any one knows why this problems comes just clear my concept about it.

The problem is not in "node_modules/react-native-reanimated" but in the "node_modules/react-native".

Boost is a portable C++ source libraries need to the compilation :

https://www.boost.org

**Comment by passiondroid** [comment]
Looks like this problem started happening globally after 29th Dec.
Temporary solution is to create a patch file for now.

**Comment by ssommelet21** [comment]
> Looks like this problem started happening globally after 29th Dec. Temporary solution is to create a patch file for now.

No it works without path now.
Server is back you can test some download for all version here : https://www.boost.org/users/history/

**Comment by h-assefi** [comment]
is it resolved?

**Comment by h-assefi** [comment]
> For Android, one quick workaround is to create patch-package for package react-native-reanimated.
> 
> Just replace
> 
> ```
>     task downloadBoost(dependsOn: resolveBoost, type: Download) {
>         def transformedVersion = BOOST_VERSION.replace("_", ".")
>         def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
>         def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
>         if (REACT_NATIVE_MINOR_VERSION < 69) {
>             srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
>         }
>         src(srcUrl)
>         onlyIfNewer(true)
>         overwrite(false)
>         dest(new File(downloadsDir, artifactLocalName))
>     }
> ```
> 
> in `/node_modules/react-native-reanimated/android/build.gradle` with
> 
> ```
> task downloadBoost(dependsOn: resolveBoost, type: Download) {
>         def transformedVersion = BOOST_VERSION.replace("_", ".")
>         def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
>         
>         def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"
>         if (REACT_NATIVE_MINOR_VERSION < 69) {
>             srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
>         }
>         src(srcUrl)
>         onlyIfNewer(true)
>         overwrite(false)
>         dest(new File(downloadsDir, artifactLocalName))
>     }
> ```
> 
> This should fix the android build issue for timebeing. @Temitope-Emmanuel
> 
> _Make sure to delete the previously downloaded file boost_1_76_0.tar.gz in react-native-reanimated/android/build/downloads_

I am using EAS to do the build production. 
How can I do this?

**Comment by swaruphs** [comment]
seems the jfrog url is working now.

**Comment by anirudh-antino** [comment]
**Temporary fix:** 
You can download `boost_1_76_0.tar.gz` file from here https://source.ipfire.org/source-2.x/
In your case you can download it via this link: https://source.ipfire.org/source-2.x/boost_1_76_0.tar.gz

then just copy it to `App/node_modules/react-native-reanimated/android/build/downloads/boost_1_76_0.tar.gz `and build your app.

Note: you need to do this every time you run `yarn` or `npm`. I usually do this since it downloads this file every time and jfrog server is very very slow to download.

**Comment by h-assefi** [comment]
it seems, its working now.

**Comment by abed-obaah** [comment]
Yes it's working now

**Comment by cortinico** [comment]
This can be closed as is effectively a duplicate of https://github.com/facebook/react-native/issues/42109

**Comment by chandresh8187** [comment]
i changed 
[def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"] 
To 
[def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"]

in my App/node_modules/react-native-reanimated/android/build.gradle

and it works

also it is work for expo-modules-core same issue

**Comment by Linuhusainnk** [comment]
Patching 

Reanimated(android) and react-native(for IOS) works

**Comment by armostheus** [comment]
Update: Started getting this error again today (7th Jan 2024)

**Comment by hridoy43** [comment]
Getting the same error from yesterday.  It's fine when trying to build the android app from linux. However, encountered the error from M1

**Comment by mykelFITLG** [comment]
also experience the same issue now

**Comment by md-irefer** [comment]
same issue here in react-native 0.69.4 (expo 46.0.3), both in M1 and windows

**Comment by aryanagarwal15** [comment]
Same issue occurring again

**Comment by md-irefer** [comment]
can we reopen this ticket?

**Comment by aryanagarwal15** [comment]
This issue is occuring again

**Comment by brunofariasd** [comment]
Same error here today (7th Jan 2024)

**Comment by wizarddev038** [comment]
It's been for 10 times since I started building the app today and it says I can't fetch that jfrog url package. How can I solve this issue? I'm using "eas build" command to build the app and the platform is android.

**Comment by wizarddev038** [comment]
> Having the same issue 7th Jan 2024

How long has it been? I face this issue from 10 hours ago.

**Comment by favianCVM** [comment]
same here. Trying to build a detox test build

**Comment by compwron** [comment]
same

**Comment by nigelrudolf** [comment]
As mentioned above,

In `/node_modules/react-native-reanimated/android/build.gradle`


Change this:
```
def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
```
To This:
```
def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"
```

Make sure to delete the previously downloaded file `boost_1_76_0.tar.gz` in `react-native-reanimated/android/build/downloads`

**Comment by md-irefer** [comment]
my error is not in react-native-reanimated, but in expo-modules-core
* What went wrong:
Execution failed for task ':expo-modules-core:prepareBoost'.
> Could not read .../node_modules/expo-modules-core/android/build/downloads/boost_1_76_0.tar.gz.
   > Not in GZIP format

**Comment by Tushar-Komprise** [comment]
same issue

**Comment by brianwestphal** [comment]
Raised https://github.com/boostorg/boost/issues/849

**Comment by dsumayan21** [comment]
same here. react native v 0.69.9

![image](https://github.com/software-mansion/react-native-reanimated/assets/96033769/c8a25cfd-49c7-4085-bb09-d70b20c0dc43)

**Comment by imamrobani** [comment]
> As mentioned above,
> 
> In `/node_modules/react-native-reanimated/android/build.gradle`
> 
> Change this:
> 
> ```
> def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
> ```
> 
> To This:
> 
> ```
> def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"
> ```
> 
> Make sure to delete the previously downloaded file `boost_1_76_0.tar.gz` in `react-native-reanimated/android/build/downloads`

temporary it's work, thanks!

**Comment by wizarddev038** [comment]
> > As mentioned above,
> > In `/node_modules/react-native-reanimated/android/build.gradle`
> > Change this:
> > ```
> > def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > To This:
> > ```
> > def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Make sure to delete the previously downloaded file `boost_1_76_0.tar.gz` in `react-native-reanimated/android/build/downloads`
> 
> temporary it's work, thanks!

How do I set it in expo? I think when running "eas build", all the dependencies are uploaded to the cloud server and download packages automatically so I can't change the code or values in the packages. How can I do it?

**Comment by ergulemre** [comment]
it is solved [Migrate boost download away from JFrog ](https://github.com/facebook/react-native/pull/42118/files)

**Comment by wizarddev038** [comment]
Is it fixed for expo-module-core too?

**Comment by wizarddev038** [comment]
![image](https://github.com/software-mansion/react-native-reanimated/assets/129702384/72ca7047-b807-444f-9415-ab9c4c336f85)
![image](https://github.com/software-mansion/react-native-reanimated/assets/129702384/99903925-6d02-48bc-99cb-0d8f49dd27fb)
Still the same issue. Please help me with this.

**Comment by passiondroid** [comment]
As mentioned above,
In /node_modules/expo-modules-core/android/build.gradle
Change this:

def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
To This:

def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"

And then create a patch file using this command -
`npx patch-package expo-modules-core`

Temporary workaround. Working for me

**Comment by Jean-Bekker** [comment]
Fixed the Issue with Boost Library

Step 1: Update URL in ReactAndroid Build Script

1. Navigate to the file: `/node_modules/react-native/ReactAndroid/build.gradle`

2. Locate the following line:

```
https://boostorg.jfrog.io/artifactory/main/release/${BOOST_VERSION.replace("_", ".")}/source/boost_${BOOST_VERSION}.tar.gz
```

3. Replace it with:

```
https://archives.boost.io/release/${BOOST_VERSION.replace("_", ".")}/source/boost_${BOOST_VERSION}.tar.gz
```

Step 2: Update URL in Expo Modules Core

1. Go to the file: `/node_modules/expo-modules-core/android/build.gradle.`

2. Search for the same URL as in Step 1.

3. Replace it with the new URL provided in Step 1.

Step 3: Apply the Patches

After making the changes, run the following commands to apply the patches:
```
patch-package react-native
patch-package expo-modules-core
```

**Comment by trickeyd** [comment]
I applied the patch to:
- react-native
- react-native-reanimated
- expo-modules-core

Now I get this error:
`
* What went wrong:
Execution failed for task ':react-native-reanimated:unzipHermes'.
> Could not read /Users/jamestrickey/projects/goss-mobile-app/node_modules/react-native/sdks/download/hermes.tar.gz.
   > java.io.EOFException (no error message)
`

Not sure how hermes was effected. I have deleted node_modules and cleared the android build cache but to no avail. Anyone else seen this?

**Comment by marc-sarroca-thepowermba** [comment]
same issue here trying to generate builds in AppCenter

**Comment by sauravjmedhi-softsensor** [comment]
> I applied the patch to:
> 
> * react-native
> * react-native-reanimated
> * expo-modules-core
> 
> Now I get this error:
> 
> ```
> * What went wrong:
> Execution failed for task ':react-native-reanimated:unzipHermes'.
> > Could not read /Users/***/node_modules/react-native/sdks/download/hermes.tar.gz.
>    > java.io.EOFException (no error message)
> ```
> 
> Not sure how hermes was effected. I have deleted node_modules and cleared the android build cache but to no avail. Anyone else seen this?

A solution given in another issue seems to work: https://github.com/software-mansion/react-native-reanimated/issues/3590#issuecomment-1255475367

But, how to make this change so that it works in a ci/cd as well?

**Comment by Marcuspo** [comment]
same

**Comment by lucianomlima** [comment]
I'm getting error for iOS too, during pod install commando in App Center. What I need to change before creating the patch?
```
Installing boost (1.76.0)

[!] Error installing boost
Verification checksum was incorrect, expected f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41, got 5e89103d9b70bba5c91a794126b169cb67654be2051f90cf7c22ba6893ede0ff

##[error]The process '/usr/local/lib/ruby/gems/3.0.0/bin/pod' failed with exit code 1
##[error]The 'pod' command failed with error: The process '/usr/local/lib/ruby/gems/3.0.0/bin/pod' failed with exit code 1
##[section]Finishing: Pod install
```

**Comment by SYoder1** [comment]
@lucianomlima React Native posted a workaround to fix the ios error. https://github.com/facebook/react-native/issues/42180#issue-2070192742

**Comment by SYoder1** [comment]
> I'm getting error for iOS too, during pod install commando in App Center. What I need to change before creating the patch?
> 
> ```
> Installing boost (1.76.0)
> 
> [!] Error installing boost
> Verification checksum was incorrect, expected f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41, got 5e89103d9b70bba5c91a794126b169cb67654be2051f90cf7c22ba6893ede0ff
> 
> ##[error]The process '/usr/local/lib/ruby/gems/3.0.0/bin/pod' failed with exit code 1
> ##[error]The 'pod' command failed with error: The process '/usr/local/lib/ruby/gems/3.0.0/bin/pod' failed with exit code 1
> ##[section]Finishing: Pod install
> ```

@lucianomlima React Native posted a workaround to fix the ios error. [facebook/react-native#42180 (comment)](https://github.com/facebook/react-native/issues/42180#issue-2070192742)

**Comment by dsumayan21** [comment]
> Fixed the Issue with Boost Library
> 
> Step 1: Update URL in ReactAndroid Build Script
> 
> 1. Navigate to the file: `/node_modules/react-native/ReactAndroid/build.gradle`
> 2. Locate the following line:
> 
> ```
> https://boostorg.jfrog.io/artifactory/main/release/${BOOST_VERSION.replace("_", ".")}/source/boost_${BOOST_VERSION}.tar.gz
> ```
> 
> 3. Replace it with:
> 
> ```
> https://archives.boost.io/release/${BOOST_VERSION.replace("_", ".")}/source/boost_${BOOST_VERSION}.tar.gz
> ```
> 
> Step 2: Update URL in Expo Modules Core
> 
> 1. Go to the file: `/node_modules/expo-modules-core/android/build.gradle.`
> 2. Search for the same URL as in Step 1.
> 3. Replace it with the new URL provided in Step 1.
> 
> Step 3: Apply the Patches
> 
> After making the changes, run the following commands to apply the patches:
> 
> ```
> patch-package react-native
> patch-package expo-modules-core
> ```

This worked for me. Thanks for the temp fix!

**Comment by SHINGPH** [comment]
> For Android, one quick workaround is to create patch-package for package react-native-reanimated.
> 
> Just replace
> 
> ```
>     task downloadBoost(dependsOn: resolveBoost, type: Download) {
>         def transformedVersion = BOOST_VERSION.replace("_", ".")
>         def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
>         def srcUrl = "https://boostorg.jfrog.io/artifactory/main/release/${transformedVersion}/source/boost_${BOOST_VERSION}.tar.gz"
>         if (REACT_NATIVE_MINOR_VERSION < 69) {
>             srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
>         }
>         src(srcUrl)
>         onlyIfNewer(true)
>         overwrite(false)
>         dest(new File(downloadsDir, artifactLocalName))
>     }
> ```
> 
> in `/node_modules/react-native-reanimated/android/build.gradle` with
> 
> ```
> task downloadBoost(dependsOn: resolveBoost, type: Download) {
>         def transformedVersion = BOOST_VERSION.replace("_", ".")
>         def artifactLocalName = "boost_${BOOST_VERSION}.tar.gz"
>         
>         def srcUrl = "https://sourceforge.net/projects/boost/files/boost/${transformedVersion}/boost_${BOOST_VERSION}.tar.gz"
>         if (REACT_NATIVE_MINOR_VERSION < 69) {
>             srcUrl = "https://github.com/react-native-community/boost-for-react-native/releases/download/v${transformedVersion}-0/boost_${BOOST_VERSION}.tar.gz"
>         }
>         src(srcUrl)
>         onlyIfNewer(true)
>         overwrite(false)
>         dest(new File(downloadsDir, artifactLocalName))
>     }
> ```
> 
> This should fix the android build issue for timebeing. @Temitope-Emmanuel
> 
> _Make sure to delete the previously downloaded file boost_1_76_0.tar.gz in react-native-reanimated/android/build/downloads_

i have tried this fix but now i am having problem with 
and now i have downloaded the boost 1_76_0 on sourceforge website and added it to appname/node_modules/react-native-reanimated/android/build/downloads/boost_1_76_0.tar.gz. and now here is the error.

/node_modules/react-native/ReactCommon/cxxreact/CxxModule.h:15:10: fatal error: 'folly/dynamic.h' file not found
#include <folly/dynamic.h>
^~~~~~~~~~~~~~~~~
1 error generated.

**Comment by h-assefi** [comment]
> Fixed the Issue with Boost Library
> 
> Step 1: Update URL in ReactAndroid Build Script
> 
> 1. Navigate to the file: `/node_modules/react-native/ReactAndroid/build.gradle`
> 2. Locate the following line:
> 
> ```
> https://boostorg.jfrog.io/artifactory/main/release/${BOOST_VERSION.replace("_", ".")}/source/boost_${BOOST_VERSION}.tar.gz
> ```
> 
> 3. Replace it with:
> 
> ```
> https://archives.boost.io/release/${BOOST_VERSION.replace("_", ".")}/source/boost_${BOOST_VERSION}.tar.gz
> ```
> 
> Step 2: Update URL in Expo Modules Core
> 
> 1. Go to the file: `/node_modules/expo-modules-core/android/build.gradle.`
> 2. Search for the same URL as in Step 1.
> 3. Replace it with the new URL provided in Step 1.
> 
> Step 3: Apply the Patches
> 
> After making the changes, run the following commands to apply the patches:
> 
> ```
> patch-package react-native
> patch-package expo-modules-core
> ```

This worked for me. Thanks for the fix!

**Comment by Jahanzeb009** [comment]
Install this package,

> `expo-modules-core`

This solved my problem.

**Comment by marc-sarroca-thepowermba** [comment]
Seems that is working now again, I tried to do some builds in AppCenter in IOS and Android and works fine!

**Comment by piaskowyk** [comment]
tldr: this problem was caused by unavailable jFrog servers services, the solution is to change a just a dependency repository URL like here: https://github.com/software-mansion/react-native-reanimated/issues/5542#issuecomment-1880844228 or just wait a couple minutes. More info here: https://github.com/software-mansion/react-native-reanimated/issues/5542#issuecomment-1873833655
